mitosheet/mitosheet/tests/saved_analyses/test_save_utils.py

```diff
@@ -281,6 +281,23 @@ def test_upgrades_old_analysis_before_replaying_it():
         pd.DataFrame({'A': [123], 'B_sum': [123]})
     )
 
+def test_save_analysis_has_code():
```

Comment:
Yeah -- we just need a test that an old saved analysis (you can get this from your `.mito/saved_analyses` folder) upgrades to have a code key -- so we don't access it somewhere by accident.